src/SimpleSimulator.cpp

```diff
@@ -140,7 +154,7 @@ po::variables_map process_program_options(int argc, char** argv) {
 
         ("duplications,d", po::value<size_t>()->default_value(duplications), "number of duplications of the workflow to feed into the simulation")
 
-        ("no-streaming", po::bool_switch()->default_value(no_blockstreaming), "switch to turn on/off block-wise streaming of input-files")
+        ("workflow-type", po::value<std::string>()->default_value("streaming"), "switch to define the type of the workflow. Please choose from 'calculation', 'streaming', or 'copy'")
```

Comment:
You could restrict the choices here in code, like done for the `cacheScope`.

Reply:
I thought since this is already checked by the [translator function](https://github.com/HEPCompSim/DCSim/blob/feature/multiple-workflows-per-json/src/util/Enums.h#L15-L28), I could neglect it here. But if you like I can extend it with a catch of the names, too.

Reply:
I guess it would be already helpful for the user to be already restricted at this point.

Reply:
Done: https://github.com/HEPCompSim/DCSim/pull/41/commits/40a9a26110f2b4dcd3024fc2a112b8354357f742